src/transaction_builder.js

```diff
@@ -135,6 +137,21 @@ function TransactionBuilder (network) {
   this.tx = new Transaction()
 }
 
+TransactionBuilder.setLockTime = function (locktime) {
+  typeforce(types.Uint32, locktime)
+
+  // if any signatures exist, throw
+  if (this.inputs.some(function (input) {
+    if (!input.signatures) return false
+
+    return input.signatures.some(function (s) { return s })
+  })) {
+    throw new Error('No, this would invalidate signatures')
```

Comment:
I still find this error message funny because it answers "no" without any question asked :P